tests/suites/1_stateful/05_formats/05_06_json/05_06_01_json_output.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
```

Comment:
Can we test this via sqllogictest?

Reply:
does sqllogictest support bash?

Reply:
we need to use this featur, let merge it first.